addons/account/wizard/account_move_reversal.py

```diff
@@ -131,7 +131,9 @@ def reverse_moves(self, is_modify=False):
             if is_modify:
                 moves_vals_list = []
                 for move in moves.with_context(include_business_fields=True):
-                    moves_vals_list.append(move.copy_data({'date': self.date})[0])
+                    data = move.copy_data({'date': self.date})[0]
+                    data['line_ids'] = [line for line in data['line_ids'] if line[2]['display_type'] == 'product']
```

Comment:
I think that part is wrong :thinking: Have a miscellaneous operations all its lines set as `product` ?

Reply:
I changed it to be `if line[2]['display_type'] != 'payment_term'`

The reason is that since we are changing the date, the date of the payment terms need to be recomputed as well.

You will never have payment terms on misc entries.
The taxes will be conserved thanks to the lines added in `_sync_dynamic_line`